script.js

```diff
@@ -0,0 +1,201 @@
+// ====== html 요소 변수 선언  ======= //
+
+const form = document.querySelector(".formControl");
+const formInput = document.querySelector(".formInput");
+const submitBtn = document.querySelector(".submitBtn");
+const todoList = document.querySelector(".todoList");
+const doneList = document.querySelector(".doneList");
+const todoCountField = document.querySelector(".todoCount");
+const doneCountField = document.querySelector(".doneCount");
+
+// form | form 요소
+// formInput | form 입력 영역 요소
+// submitBtn | form 내 등록 버튼
+// todoList | todoItem 가지는 list container
+// doneList | doneItem 가지는 list container
+// todoCountField, doneCountField | 각 count 값 가지는 text container
+
+// ====== 초기화시 실행 동작 ======= //
+
+window.addEventListener("DOMContentLoaded", () => {
+  updateCnt();
+  let items = getLocalStorage();
+
+  if (items.length > 0) {
+    items.forEach(function (item) {
+      if (item.value.type === "todo") {
+        createTodoItem(item.id, item.value.text);
+      } else if (item.value.type === "done") {
+        createDoneItem(item.id, item.value.text);
+      }
+    });
+  }
+
+  // LocalStorage에서 Item을 불러오고, 비어있는 경우 예외
+});
```

Comment:
현재 로컬스토리지에서 값을 불러온 후 개수가 제대로 카운팅되지 않는 것 같습니다. 아래와같이 수정하시면 정상적으로 카운팅됩니다!

```suggestion
window.addEventListener("DOMContentLoaded", () => {
  let items = getLocalStorage();

  if (items.length > 0) {
    items.forEach(function (item) {
      if (item.value.type === "todo") {
        createTodoItem(item.id, item.value.text);
      } else if (item.value.type === "done") {
        createDoneItem(item.id, item.value.text);
      }
    });
  }

  updateCnt();
  // LocalStorage에서 Item을 불러오고, 비어있는 경우 예외
});
```

Reply:
대헌님 코드도 좋다 생각했는데 리뷰해주신 부분도 좋은 것 같아서, 글 읽으면서 배워 갑니당 👍

Reply:
맞아요, updateCnt()가 뒤에 들어가야겠군요. 감사합니다!